### PR TITLE
Add registration scenario

### PR DESCRIPTION
**Pull request by kmatlaha**
Add registration scenario

### CODE REVIEW

File: tests/store_test.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+Scenario('test something', ({ I, homePage, registerPage }) => {
+    I.openStore();
+    homePage.openRegistrationPage();
+    registerPage.verifyRegisterAccountText();
+    registrationUser.email = Date.now() + '@test.com';
+    registerPage.fillRegistrationDetails(registrationUser);
+    registerPage.clickSubscribeYesButton();
+    registerPage.clickAgreePolicyButton();
+    registerPage.clickContinueOnRegisterPageButton();
+    I.see('Your Account Has Been Created!');
```
Comment (adudyak) [inline tests/store_test.js:20]: use your method in register.js

File: tests/store_test.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+    pause();
```
Comment (adudyak) [inline tests/store_test.js:21]: remove pause()

File: tests/store_test.js [hunk 3]
```diff
@@ -0,0 +1,22 @@
+let registrationUser = {
+    firstName: 'Kate',
+    lastName: '3',
+    telephoneNumber: '+38096578965',
+    password: 'Temp12345',
+    passwordConfirm: 'Temp12345',
```
Comment (adudyak) [inline tests/store_test.js:6]: remove passwordConfirm, use password

File: steps_file.js [hunk 4]
```diff
@@ -0,0 +1,14 @@
+// in this file you can append custom step methods to 'I' object
+const STORE_URL = 'http://opencart.qatestlab.net/index.php';
+
+module.exports = function() {
+  return actor({
+    openStore() {
+      this.amOnPage (STORE_URL);
```
Comment (adudyak) [inline steps_file.js:7]: remove space this.amOnPage(STORE_URL);

**Review by adudyak — CHANGES_REQUESTED** [review]

File: tests/store_test.js (hunk 1 above)
Comment (kmatlaha) [reply tests/store_test.js:20]: fixed

File: tests/store_test.js (hunk 2 above)
Comment (kmatlaha) [reply tests/store_test.js:21]: fixed

File: tests/store_test.js (hunk 3 above)
Comment (kmatlaha) [reply tests/store_test.js:6]: fixed

File: steps_file.js (hunk 4 above)
Comment (kmatlaha) [reply steps_file.js:7]: fixed

**Pull request closed without merging**